exporter/zipkinexporter/factory.go

```diff
@@ -56,13 +56,13 @@ func (f *Factory) CreateTraceExporter(logger *zap.Logger, config configmodels.Ex
 	if cfg.URL == "" {
 		return nil, nil, errors.New("exporter config requires a non-empty 'url'") // TODO: better error
 	}
-
+	// TODO Fix service name. Should this be the full name of the exporter?
```

Comment:
This is used if the zipkin span is not carrying the name of the service, which shouldn't happen in normal circumstances only due to (bad) conversions between formats. The current value is a clear indication that somehow the name of the service was lost in translation.

Reply:
I'm going to take your comment and add it to the code base for everyone to get the context.
The todo will be removed.